.env.example

```diff
@@ -1,5 +1,6 @@
 NODE_ENV=development
 PORT=4040
-JWT_SECRET=0a6b944d-d2fb-46fc-a85e-0295c986cd9f
 MONGO_HOST=mongodb://localhost/hl7-telescoper-api-server-development
 MONGO_PORT=27017
+JWT_SECRET=0a6b944d-d2fb-46fc-a85e-0295c986cd9f
+JWT_EXP_TIME=24h
```

Comment:
Let's set this to a safe default like `1hr` (I see you did that in the Joi config, but there's a risk that this example `.env` will be copy/pasted and not looked at closely).

Reply:
haha, this wasn't meant to make it here. I did 24h on my .env and forgot to change it here